pkg/cloud/openstack/machine/actuator.go

```diff
@@ -66,6 +68,16 @@ func NewActuator(params openstack.ActuatorParams) (*OpenstackClient, error) {
 	}, nil
 }
 
+func getEnv(name string, def int) time.Duration {
```

Comment:
The function is called 'getEnv' but it's very timeout specific. It should at least be renamed to getTimeout or something like that.